src/components/sbb-slider/sbb-slider.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  JSX,
+  Prop,
+  State,
+} from '@stencil/core';
+import { hostContext } from '../../global/helpers/host-context';
+import { AccessibilityProperties } from '../../global/interfaces/accessibility-properties';
+import { SbbSliderChange } from './sbb-slider.custom';
+
+/**
+ * @slot prefix - Slot to render an icon on the left side of the input.
+ * @slot suffix - Slot to render an icon on the right side of the input.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-slider.scss',
+  tag: 'sbb-slider',
+})
+export class SbbSlider implements ComponentInterface, AccessibilityProperties {
+  /** Value for the inner HTMLInputElement. */
+  @Prop() public value?: string = '';
+
+  /** Numeric value for the inner HTMLInputElement. */
+  @Prop() public valueAsNumber?: number;
+
+  /** Name of the inner HTMLInputElement. */
+  @Prop() public name?: string = '';
+
+  /** The <form> element to associate the inner HTMLInputElement with. */
+  @Prop() public form?: string;
+
+  /** Minimum acceptable value for the inner HTMLInputElement. */
+  @Prop() public min?: string = '0';
+
+  /** Maximum acceptable value for the inner HTMLInputElement. */
+  @Prop() public max?: string = '100';
+
+  /**
+   * Readonly state for the inner HTMLInputElement.
+   * Since the input range does not allow this attribute, it will be merged with the `disabled` one.
+   */
+  @Prop() public readonly?: boolean = false;
+
+  /** Disabled state for the inner HTMLInputElement. */
+  @Prop() public disabled?: boolean = false;
+
+  /** Name of the icon at component's start, which will be forward to the nested `sbb-icon`. */
+  @Prop() public startIcon?: string;
+
+  /** Name of the icon at component's end, which will be forward to the nested `sbb-icon`. */
+  @Prop() public endIcon?: string;
+
+  /** This will be forwarded as aria-label to the relevant nested element. */
+  @Prop() public accessibilityLabel: string | undefined;
+
+  /** This will be forwarded as aria-describedby to the relevant nested element. */
+  @Prop() public accessibilityDescribedby: string | undefined;
+
+  /** This will be forwarded as aria-labelledby to the relevant nested element. */
+  @Prop() public accessibilityLabelledby: string | undefined;
+
+  /** Event emitted when the value of the inner HTMLInputElement changes. */
+  @Event() public sbbChange: EventEmitter<SbbSliderChange>;
```

Comment:
I'm inclined to drop custom form events (for change and input), similar to what we decided for action elements.
@jeripeierSBB, @DavideMininni-Fincons, @simone-sabato Thoughts?

Reply:
I agree, but most of these values could also be read from the target element.

Reply:
It was not that easy as the target becomes the sbb-slider. but now we update the value there as well, so it's readable from outside.